docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing Octopus issues
+
+This repository is used by the Octopus team to communicate with our community about what we are working on for the core Octopus Server product.
+
+## Do not create issues here directly
+
+If you've found a bug or something isn't working, please get in touch with [our support team](https://octopus.com/support) instead. We don't read new GitHub issues regularly, so if you post here it will likely not get a reply. If you [contact support](https://octopus.com/support), we can probably help to find a workaround or make sure the issue gets prioritized properly.

```

Comment:
```suggestion
If you've found a bug or something isn't working, please get in touch with [our support team](https://octopus.com/support). We don't read new GitHub issues regularly, so if you post here it will likely not get a reply. If you [contact support](https://octopus.com/support), we can probably help to find a workaround or make sure the issue gets prioritized properly.

```